examples/using_fhir_views.ipynb

```diff
@@ -0,0 +1,671 @@
+       "      <td>[\"01562\"]</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>4768</td>\n",
+       "      <td>female</td>\n",
+       "      <td>1999-02-12</td>\n",
+       "      <td>[\"508 Auer Lodge\"]</td>\n",
+       "      <td>[\"Wilbraham\"]</td>\n",
+       "      <td>[\"MA\"]</td>\n",
+       "      <td>[]</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>4770</td>\n",
+       "      <td>male</td>\n",
+       "      <td>1999-07-01</td>\n",
+       "      <td>[\"230 Olson Fort Suite 50\"]</td>\n",
+       "      <td>[\"Dennis\"]</td>\n",
+       "      <td>[\"MA\"]</td>\n",
+       "      <td>[]</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>4776</td>\n",
+       "      <td>female</td>\n",
+       "      <td>1993-07-31</td>\n",
+       "      <td>[\"612 Kuhlman Skyway Apt 45\"]</td>\n",
+       "      <td>[\"Amherst\"]</td>\n",
+       "      <td>[\"MA\"]</td>\n",
+       "      <td>[]</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     id  gender  birthdate                         street             city  \\\n",
+       "0  4765    male 1984-09-09    [\"754 Feil Tunnel Unit 36\"]  [\"Springfield\"]   \n",
+       "1  4767  female 1999-04-26  [\"766 King Landing Suite 14\"]      [\"Spencer\"]   \n",
+       "2  4768  female 1999-02-12             [\"508 Auer Lodge\"]    [\"Wilbraham\"]   \n",
+       "3  4770    male 1999-07-01    [\"230 Olson Fort Suite 50\"]       [\"Dennis\"]   \n",
+       "4  4776  female 1993-07-31  [\"612 Kuhlman Skyway Apt 45\"]      [\"Amherst\"]   \n",
+       "\n",
+       "    state        zip  \n",
+       "0  [\"MA\"]  [\"01119\"]  \n",
+       "1  [\"MA\"]  [\"01562\"]  \n",
+       "2  [\"MA\"]         []  \n",
+       "3  [\"MA\"]         []  \n",
+       "4  [\"MA\"]         []  "
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pandas.read_sql_query(\n",
+    "    sql=\"SELECT * FROM statin_analysis_example.patients LIMIT 5\",\n",
+    "    con=query_engine,\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we have a nice, flattened patients table that meets the needs of our system and data."
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.9"
+  },
+  "orig_nbformat": 4
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
Is it easy to expose the underlying SQL query that the FHIR-views library generates? If yes, then please add 1-2 examples to show some complex SQL queries generated by the library.

Reply:
Is this addressed? Unresolving for now as I don't see examples of produced SQL in the latest version.